### DISCUSSION TITLE
Gta 5 crashes when I press f4 to fire up the console while playing the game

### DISCUSSION THREAD

**Discussion by Berlinowski** [scripthookvdotnet/scripthookvdotnet#1538] [General]
I reinstalled gta 5, and to that scripthookV and scripthookVdoNet and as soon as I enter the game and try to turn on the console gta stops and shuts down after a while.

Last what i had in the logs:
[ScriptHookVDotNet.log](https://github.com/user-attachments/files/18269128/ScriptHookVDotNet.log)

**Comment by kagikn** [comment]
You encountered the infamous compatibility problem resolved in later nightly versions (see #1451 for details). Use a nightly version from [the releases on the nightly release repo](https://github.com/scripthookvdotnet/scripthookvdotnet-nightly/releases) for now. After v3.7.0 gets released, you'll be able to use it (or later stable versions) without encountering that problem.
